gammapy/image/mask.py

```diff
@@ -95,8 +93,28 @@ def plot(self, ax=None, fig=None, **kwargs):
 
     @lazyproperty
     def distance_image(self):
-        """Map containting the distance to the nearest exclusion region."""
-        return exclusion_distance(self.mask)
+        """Distance to nearest exclusion region.
+
+        Compute distance map, i.e. the Euclidean (=Cartesian 2D)
+        distance (in pixels) to the nearest exclusion region.
+
+        We need to call distance_transform_edt twice because it only computes
+        dist for pixels outside exclusion regions, so to get the
+        distances for pixels inside we call it on the inverted mask
+        and then combine both distance images into one, using negative
+        distances (note the minus sign) for pixels inside exclusion regions.
+
+        Returns
+        -------
+        distance : `~numpy.ndarray`
```

Comment:
We decided to return a `SkyMap` in the GSoC call just now.

My preference would be to implement https://github.com/gammapy/gammapy/issues/632 and then add an option here that returns the sky map in degree, where the pixel distance image is multiplied by `proj_plane_pixel_scales`. But that's for a future PR, we don't have to discuss here.